lessons/Early-Math/associative-property/langs/en-us/pages/assoc-prop-table.adoc

```diff
@@ -0,0 +1,49 @@
+= Associative Property Table
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+
+For each Circle of Evaluation, apply the Associative Property to create two equivalent Circles. Make sure you use different _groupings_ in each Circle. Note: There are multiple possible responses here! @ifsoln{Some possible responses are shown below. Your students will likely come up with others.}
```

Comment:
#3 and #6 are much more advanced questions and I don't think we've provided the scaffolding for them! That doesn't mean I don't think they're important, but they may have to wait for an expansion of early math materials. I don't remember anything in this pathway discussing order of operations in a way that would support students in understanding the very limited options they have for regrouping these expressions based on the hierarchy of the structure... a particular risk because we are explicitly asking them to blow apart some of the structure. Am I wrong?

Reply:
In the commutative property lesson, there are problems like this as well - with multiple operations.

This is well within the realm of what students in grades 5-7 should be able to do. I want our stuff to be accessible, but I also want it to be rigorous.

I think there is adequate support. The fact that the associative property does not hold for division and subtraction is emphasized throughout. Kids can deduce accordingly, especially with the help of teachers / peers.  There is even a note in the lesson calling out these problems and recommending that teachers discuss them with kids.

I want our materials to be accessible, but I am concerned about watering things down. Kids in grades 5-7 could very well stumble when faced with these problems but overall should be equipped to engage and grow.

Relatedly, I've also occasionally included accessible fractions / decimals at the end of worksheets (e.g., fractions that add up to 1, decimals in the tenths place). Realistically, kids in grades 5-7 should be getting *regular* exposure to fractions and decimals. I don't think we're doing students any favors by eliminating them completely.